.travis.yml

```diff
@@ -14,11 +16,16 @@ before_install:
   - cp config/secrets.travis.yml config/secrets.yml
   - git submodule init
   - git submodule update --depth=10 -- red5
+  - rvm use 2.3.3@global
+  # Bundler 1.15 has stricter dependency checking -- problematic for Hydra projects
+  - gem uninstall -aIx bundler
+  - gem install bundler -v 1.14.6
   - bundle config without development:production:staging
```

Comment:
I think you misread, or I'm confused. There is a "gem install bundle" not a "bundle install" (maybe I didn't understand this comment).